includes/languages/english/extra_definitions/bootstrap/zca_bootstrap_common.php

```diff
@@ -1,6 +1,75 @@
 <?php
 // -----
-// Part of the Bootstrap template, defining commonly-used phrases.
+// Part of the Bootstrap template, defining commonly-used phrases and phrases unique to the bootstrap template.
 //
 define('BOOTSTRAP_PLEASE_SELECT', 'Please select ...');
 define('BOOTSTRAP_CURRENT_ADDRESS', ' (Currently Selected)');
+
+// -----
+// Additional buttons.
+//
+define('BUTTON_IMAGE_ADD_TO_CART', 'button_add_to_cart.gif');
+define('BUTTON_IMAGE_MY_ACCOUNT', 'button_my_account.gif');
+
+define('BUTTON_ADD_TO_CART_ALT', 'Add This to My Cart');
+define('BUTTON_MY_ACCOUNT_TEXT', 'My Account');
+
+define('BUTTON_RETURN_TO_PROD_LIST_ALT', 'Product Listing');
+
+// -----
+// Used during checkout and on various address-rendering pages.
+//
+define('TEXT_SELECT_OTHER_PAYMENT_DESTINATION', 'Please select the preferred billing address if the invoice to this order is to be delivered elsewhere.');
+define('TEXT_SELECT_OTHER_SHIPPING_DESTINATION', 'Please select the preferred shipping address if this order is to be delivered elsewhere.');
+define('NEW_ADDRESS_TITLE', 'Enter new address');
+define('TEXT_PRIMARY_ADDRESS', ' (Primary Address)');
+define('PRIMARY_ADDRESS', ' (Primary Address)');
+
+// -----
+// Used on the product*_info pages.
+//
+define('TEXT_PRODUCT_MODEL', 'Model: ');
+define('TEXT_MULTIPLE_IMAGES', ' Additional Images ');
+define('TEXT_SINGLE_IMAGE', ' Larger Image ');
+define('PREV_NEXT_FROM', ' from ');
+define('IMAGE_BUTTON_PREVIOUS','Previous Item');
+define('IMAGE_BUTTON_NEXT','Next Item');
+define('IMAGE_BUTTON_RETURN_TO_PRODUCT_LIST','Back to Product List');
+
+// -----
+// Used on the product_music_info page.
+//
+define('TEXT_ARTIST_URL', 'For more information, please visit the Artist\'s <a href="%s" target="_blank">webpage</a>.');
+define('TEXT_PRODUCT_RECORD_COMPANY', 'Record Company: ');
+
+// -----
+// Used on the shopping_cart page.
+//
+define('TEXT_CART_MODAL_HELP', '[help (?)]');
+define('HEADING_TITLE_CART_MODAL', 'Visitors Cart / Members Cart');
+
+// -----
+// Used on various pages that display the cart's contents.
+//
+define('TABLE_HEADING_QUANTITY', 'Qty.');
+define('TABLE_HEADING_MODEL', 'Model');
+define('SUB_TITLE_TOTAL', 'Total:');
+
+// -----
+// Used by various product listing pages, e.g. SNAF.
+//
+define('TEXT_PRODUCTS_MODEL', 'Model:');
+define('TABLE_HEADING_IMAGE', 'Product Image');
+define('TABLE_HEADING_MODEL', 'Model');
```

Comment:
Note that this has multiple TABLE_HEADING_MODEL (lines 55 and 63)

And in v157 `english.php` contains:
```php
// generic model
  define('TEXT_MODEL','Model:');
  define('TEXT_PRODUCTS_MODEL', 'Products Model:');
  define('TABLE_HEADING_PRODUCTS_MODEL','Model');
  define('TABLE_HEADING_MODEL', 'Model');
```